app/locales/es.json

```
@@ -9,6 +9,10 @@
     "done": "Hecho",
     "ok": "Ok"
   },
+  "dashboard": {
+    "error_title": "Un error ha ocurrido",
+    "error_message": "No se ha encontrado los datos "
```

I think this message is not user friendly, it should be some like this, same in english, if you want can make a group discussion about it  
```suggestion
    "error_title": "Oops!, se ha producido un error ",
    "error_message": "Lo sentimos!, no tenemos la informacion de la fecha especificada en nuestra base de datos, por favor pruebe con otra fecha "
```